package.json

```diff
@@ -42,4 +42,4 @@
     "prettier": "^2.2.1",
     "typescript": "^4.2.2"
   }
-}
+}
```

Comment:
Looks like we lost the `newline` here

Reply:
I think that's the action breaking it too.

Reply:
Yeah when it updates the version number, it does that - I think a PR is required to that library 🤓